### PR TITLE
rename sync variables.

### PR DESCRIPTION
**Pull request by raulk**
The `from` and `to` naming always confuses me and has me spinning every time I revisit this code. This PR introduces more specificity in the naming.

### CODE REVIEW

File: chain/sync.go [hunk 1]
```diff
@@ -1069,17 +1069,14 @@ func extractSyncState(ctx context.Context) *SyncerState {
...
+// `incoming` is the heaviest/projected/target tipset we have learned about, and
+// `known` is usually an anchor tipset we already have in our view of the chain
 // (which could be the genesis).
 //
 // collectHeaders checks if portions of the chain are in our ChainStore; falling
 // down to the network to retrieve the missing parts. If during the process, any
 // portion we receive is in our denylist (bad list), we short-circuit.
 //
-// {hint/naming}: `from` and `to` is in inverse order. `from` is the highest,
-// and `to` is the lowest. This method traverses the chain backwards.
```
Comment (schomatis) [inline chain/sync.go:1081]: > This method traverses the chain backwards.

Can we keep this? (Ignore if it is already clarified in other parts of the documentation.)

File: chain/sync.go [hunk 2]
```diff
@@ -1100,43 +1097,43 @@ func extractSyncState(ctx context.Context) *SyncerState {
...
-func (syncer *Syncer) collectHeaders(ctx context.Context, from *types.TipSet, to *types.TipSet) ([]*types.TipSet, error) {
+func (syncer *Syncer) collectHeaders(ctx context.Context, incoming *types.TipSet, known *types.TipSet) ([]*types.TipSet, error) {
 	ctx, span := trace.StartSpan(ctx, "collectHeaders")
 	defer span.End()
 	ss := extractSyncState(ctx)
 
 	span.AddAttributes(
-		trace.Int64Attribute("fromHeight", int64(from.Height())),
-		trace.Int64Attribute("toHeight", int64(to.Height())),
+		trace.Int64Attribute("fromHeight", int64(incoming.Height())),
```
Comment (schomatis) [inline chain/sync.go:1106]: nit: we may want to change trace names, unless we already depend on those names elsewhere

Comment (raulk) [reply chain/sync.go:1106]: Yeah, I didn't do this because I didn't know if we were tracking these attributes somewhere. @magik6k?

Comment (magik6k) [reply chain/sync.go:1106]: I think it's ok to rename those

**Comment by magik6k** [comment]
(rebased)

File: chain/sync.go (hunk 1 above)
Comment (magik6k) [reply chain/sync.go:1081]: `{hint/logic}` section below also explains this

**Pull request merged**